src/sources/kubernetes/mod.rs

```diff
@@ -242,20 +193,37 @@ fn transform_json_message() -> impl FnMut(Event) -> Option<Event> + Send {
                 Err(error) => warn!(message = "Non rfc3339 timestamp.", %error),
             }
         } else {
-            warn!(message = "Missing field.", field = %atom_time);
+            warn!(message = "Missing field.", field = %self.atom_time);
```

Comment:
do we want these to be warns or debugs with rate limits? I'm not sure how often these would trigger?

Reply:
Yes, all of these warns should probably be debugs with rate limits. 

They would either not trigger or trigger always, as they more or less indicate that our expectation of how `json` log from Docker looks like is wrong.

And going by that thought, it should just drop the event as other transforms, and then have `ApplicableTransform` pass event even if there are no applicable transforms. That would give users an outlet in such situations.